src/api-client.ts

```diff
@@ -2,7 +2,8 @@ import axios, { AxiosInstance } from 'axios';
 import * as FormData from 'form-data';
 import * as fs from 'fs';
 
-import RevAiAccount from './models/RevAiAccount';
+import RevAiAccount from './models/async/RevAiAccount';
+import RevAiJobOptions from './models/async/RevAiJobOptions';
```

Comment:
can we use the explicit import instead of the default import
```suggestion
import { RevAiJobOptions } from './models/async/RevAiJobOptions';
```